modules/installation-osp-default-deployment.adoc

```diff
@@ -22,19 +22,29 @@ Your quota must meet the following requirements to run the {product-title} insta
 |Subnets               | 1
 |RAM                   | 112 GB
 |vCPUs                 | 28
-|Volume storage        | 175 GB
+|Volume storage        | 275 GB
 |Instances             | 7
 |Security groups       | 3
 |Security group rules  | 60
+|================================
+
+[IMPORTANT]
+====
+If OpenStack Object Storage (Swift) is available, it is used as the default backend for the {product-title} image registry. In this case, the volume storage requirement is 175GB.
+
+.Recommend Swift resources
+[options="header"]
+|======================================
+|Resource              | Value
 |Swift containers      | 2
 |Swift objects         | 1
 |Swift available space | 10 MB or more
```

Comment:
In 4.4, we're not consuming a swift container anymore for the bootstrap ignition file. The swift containers requirement should be 1 only. Also, swift objects depend on many image are stored in the image registry AFAICT. Regarding the space, 10MB will likely not be enough to store a single image.

Since the numbers exclusively depend on the intended usage of the image-registry, I'd suggest remove this table.

See how the upstream documentation was updated in https://github.com/openshift/installer/pull/2993.

Reply:
Agreed. Will remove.